src/renderer/features/governance/aggregates/delegatedVotes.ts

```diff
@@ -10,12 +10,14 @@ import { networkSelectorModel } from '../model/networkSelector';
 import { proposerIdentityAggregate } from './proposerIdentity';
 
 const $delegatedVotesInChain = combine(
-  delegatedVotesModel.$delegatedVotes,
```

Comment:
Why you've changed so much unnecessary code?

Reply:
In lots of cases `chainId` was enough, so I've switched from `chain` to the least meaningful chunk of data.
Also I've updated `combines` so they are more readable (top to bottom) and expandable if additional data would be required. Almost from the start of effector integration me and @Asmadek decided to stick [to this rule](https://arc.net/l/quote/azxszgrz) for `combine`
Hope this explanation helps 😉